posts/2017-12-27-optimizing-final-tagless.md

```diff
@@ -0,0 +1,330 @@
+---
+layout: post
+title: Optimizing Tagless Final - Saying farewell to Free
+
+meta:
+  nav: blog
+  author: LukaJCB
```

Comment:
Please add yourself to the [authors file](https://github.com/typelevel/typelevel.github.com/blob/development/_data/authors.yml).